### DISCUSSION TITLE
nonebot无法从mirai接受到消息

### DISCUSSION THREAD

**Discussion by Arcsion** [nonebot/discussions#128] [问答] [answered]
# 问题简述
一个很蠢的问题
nonebot小白，半年前就试图把mirai接入到nonebot但是没成功，今天配置了一个下午但还是接收不到消息
表现为mirai收到消息时候nonebot端没有任何反应
## mirai端日志：
[![LdLOmT.jpg](https://s1.ax1x.com/2022/04/18/LdLOmT.jpg)](https://imgtu.com/i/LdLOmT)
## nonebot端日志：
[![LdOC11.jpg](https://s1.ax1x.com/2022/04/18/LdOC11.jpg)](https://imgtu.com/i/LdOC11)
# 相关配置文件
## mirai端mirai-api-http的setting.yml
```
adapters: 
  - ws
debug: false
enableVerify: true
verifyKey: tqjNJtYmGYmtjNpQ
singleMode: false
cacheSize: 2048
adapterSettings:
  ws:
    host: 127.0.0.1
    port: 8561
```
## nonebot端.env
```
ENVIRONMENT=dev
VERIFY_KEY=tqjNJtYmGYmtjNpQ
driver=~fastapi+~websockets

MIRAI_HOST=127.0.0.1
MIRAI_PORT=8561
MIRAI_QQ=["*********"]
SUPERUSER=["*********"]
NICKNAME=["Artius"]
COMMAND_START=["/", ""]
COMMAND_SEP=["."]
```
## nonebot端.env.dev
```
HOST=127.0.0.1
PORT=18080
LOG_LEVEL=DEBUG
FASTAPI_RELOAD=true
```
## bot.py：保持默认

8561和18080端口没有其他程序占用

**Comment by StarHeartHunt** [comment]
@ieew

**Comment by Arcsion** [comment, marked as answer]
问题已解决。解决方法：将mirai端mirai-http-api插件从v2.5.0回滚到v2.4.0。
尚不清楚具体原因，考虑提issue。

**Comment by ieew** [comment]
你的 `nonebot-adapter-mirai2` 版本号是多少？在某个版本号之前 `adapter-mirai2` 是不支持 `mirai-api-http-2.5.0` 的，如果比较老的话可以尝试更新到最新版本的 `nonebot-adapter-mirai2` 试试看。

**Reply by ieew** [reply]
截止目前， `nonebot-adapter-mirai2` 已经更新到 `0.0.11`

**Reply by Arcsion** [reply]
`nonebot-adapter-mirai2`版本为最新，已提交issue
